src/main/java/fr/zcraft/quartzlib/components/i18n/translators/gettext/PluralForms.java

```diff
@@ -173,11 +234,17 @@ private String normalizeFormsScript(final String script) {
                 return n -> n % 10 == 1 && n % 100 != 11 ? 0 : (n % 10 >= 2 && (n % 100 < 10 || n % 100 >= 20) ? 1 : 2);
 
             // Three forms, special cases for numbers ending in 1 and 2, 3, 4, except those ending in 1[1-4]
-            // Russian, Ukrainian, Belarusian, Serbian, Croatian
+            // Russian, Ukrainian, Serbian, Croatian, Belarusian, Bosnian
             case "n%10==1&&n%100!=11?0:n%10>=2&&n%10<=4&&(n%100<10||n%100>=20)?1:2":
                 return n -> n % 10 == 1 && n % 100 != 11 ? 0 :
                         (n % 10 >= 2 && n % 10 <= 4 && (n % 100 < 10 || n % 100 >= 20) ? 1 : 2);
 
+            // Alternative for the above, sometime encountered
+            // Russian, Ukrainian, Serbian, Croatian, Belarusian, Bosnian
```

Comment:
Maybe put contry name in alphebatical order `Belarusian, Bosnian, Croatian, Russian, Serbian, Ukrainian`